keyboards/gl516/j73gl/matrix.c

```diff
@@ -0,0 +1,287 @@
+/*
```

Comment:
As above.

Reply:
I tried for about 4 days after your comment on A52GL, but I could not get the input to work in Lite.
I'll leave it as "CUSTOM_MATRIX = yes".
I'll test it in the future and fix it if it works.

Reply:
Custom matrix Lite is now supported. Please confirm.